stumpy/core.py

```diff
@@ -2254,14 +2318,29 @@ def rolling_isconstant(a, w):
     w : numpy.ndarray
         The rolling window size
 
+    custom_func : object, default None
+        A custom, user-defined function that returns boolean numpy ndarray that indicate
+        if a subsequence is constant or not. It takes two arguments, `a`, a 1-D array,
+        and `w`, the window size, and may have default arguments if needed. When `None`,
+        this will be default to the function `core._rolling_isconstant`.
+
```

Comment:
I think it would be much more useful if users can easily provide a user-defined threshold for stddev. As @seanlaw mentioned before, we may consider type `float` as well. so, we may rename this parameter to something like `isconstant_criteria` that can have type `float` or be a callable object. Since calculating `Σ` is time consumting, we may add `Σ=None` and we can use it to speed up calculation when it is provied.